identity/client/src/utility/api/request.ts

```diff
@@ -99,9 +115,11 @@ const apiRequest: <R, P>(
         redirect: "manual",
       },
     );
-    if (response.status === 401) {
-      redirectToLogin();
```

Comment:
When logging out from identity, we do the following
1. Hard refresh the page
2. wait for the first 401 response
3. Load /login using a hard refresh as well

This causes the page to refresh twice.
I added a handler utility so that we can listen to the 401 instead a react component and do a FE redirect instead of doing it with a hard refresh.